app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   include Pagy::Backend
+  include Audit1984
```

Comment:
How about we include this in `Admin::BaseController` then set

```ruby
config.audits1984.base_controller_class = "::Admin::BaseController"
```
https://github.com/basecamp/audits1984#configuration